application/test/process/script_launcher_test.rb

```diff
@@ -171,7 +183,10 @@ class ScriptLauncherTest < ActiveSupport::TestCase
     # Create malformed lock file (only one line)
     File.write(@lock_file_path, '12345')
 
-    @launcher.expects(:start_process_from_script).with('scripts/launch_detached_process.rb', 'launch_detached_process.log').returns(67890)
+    expected_log_filename = 'launch_detached_process-35-W2025.log'
```

Comment:
I would use the right file name `launch_detached_process-2025-W35.log` for these tests too to avoid confusion

Reply:
Fixed